mbofui/app/js/controllers.js

```diff
@@ -1,7 +1,23 @@
 'use strict';
-/* global angular, ocellus, $, L, resolveCategory, resolveIcon, moment, validate, _ */
+/* global angular, ocellus, $, L, resolveCategory, resolveIcon, moment, validate, _, popupLink */
 
 ocellus.controller('mapController', ['$scope', '$rootScope','$filter', '$timeout', '$log', 'leafletData', 'Bof', function($scope, $rootScope, $filter, $timeout, $log, leafletData, Bof) {
+  // setting up init values
+  // setting the category filter to none
+  $scope.categories = null;
+  // setting the markers to an empty array
+  // markers array represents the filtered events
+  // markersAll represents the prefiltered events
+  $scope.markers = [];
+  $scope.markersAll = [];
+
+  // map configuration object by object:
+  // center: zoom level and autoDiscover(set to user location)
+  // TODO: need to deal with users/devices that refuse to share location
+  // events: initial event object
+  // controls: added (but not enabled: draw)
+  // layers: get baselayers (tiling options) from app.js, set overlays and draw options
+  // awesomeMarkerIcon: set a default. We will be setting different color/icons based on type of event later on
```

Comment:
Although these variable names seem relatively easy to understand, I definitely appreciate the comments.